### ISSUE TITLE
Do not copy files from cache to target when files already exist, especially videos [DONE]

### ISSUE THREAD

**Issue by jacrify** [muldjord/skyscraper#217]
**Describe the feature**
When doing reruns, filling in holes, etc, it would be great if Skyscraper checked if the target already had the resource before copying from cache. If I add one image but regenerate the whole gamelist, Skyscraper currently triggers copying of all of my videos, etc for all resources. It would be neat if there was an option to check whether the target already exists and don't copy if so.

**Comment by muldjord** [comment]
Are you aware that you, when you run Skyscraper from command-line, will be asked if you want it to skip existing gamelist entries? It's not a cli option, but one of the first things Skyscraper asks when generating the gamelist.

**Comment by jacrify** [comment]
Yes I am aware. However there is a difference between generating the gamelist and copying files from cache to target. It would be useful if there was an option to only copy files from cache to target when those files are not present at destination. 

Example use case: I had a lot of trouble with video previews on my pi before I worked out I had to adjust video memory to get them to work consistently. During debugging of this, I was regenning full gamelists with the --videos option and without. The target already had the video files in place, however each time I ran with --videos Skyscraper would kick off a full copy of my video files from cache to target. It would be neat it there was an option to only write files if not present.
